app/models/admin_only/member_app_waiting_reason.rb

```diff
@@ -0,0 +1,25 @@
+module AdminOnly
+  
+  class MemberAppWaitingReason < ApplicationRecord
+
+    validates_presence_of :name_sv
+
+    def other_reason_placeholder?
+      name_sv == self.class.other_reason_name
+    end
+
```

Comment:
We need a way to check to see if a reason is a custom reason entered by an admin, or if it's the simple placeholder that will show in the UI list.
If I add this comment before the method, will things be clear?
```
    # is this a custom reason entered or is it the placeholder for "Other (enter a custom reason)" for the UI
```

I am putting the placeholder into the db instead of reading it from a locale file (ex: `config/locale/sv.yml`).  But perhaps that's not the right design decision.  Thoughts?